lib/stemcell/launcher.rb

```diff
@@ -195,7 +195,7 @@ def launch(opts={})
         @log.info "sent ec2 api tag requests successfully"
 
         # link to classiclink
-        if @vpc_id.nil?
+        if ! @vpc_id
```

Comment:
Remove space between ! and @

Reply:
`Control flow statement with negative condition less... (⌘F1) 
This inspection warns that block with positive condition is preferable to block with negative condition`
Moved to unless

Done.